cloudmarker/stores/emailstore.py

```diff
@@ -0,0 +1,68 @@
+"""email store plugin."""
+
+
+import logging
+from email.mime.multipart import MIMEMultipart
+from email.utils import COMMASPACE, formatdate
+from smtplib import SMTP, SMTPException
+
+# Define module-level logger.
+_log = logging.getLogger(__name__)
+
+
+class EmailStore():
+    """A plugin to send email notification for anomalies found."""
+
+    def __init__(self, host, port, subject, to, sender, body):
+        """Create an instance of :class:`EmailStore` plugin.
+
+        Arguments:
+            host (string): Server hostname for SMTP.
+            port (int): Server port for STMP.
+            subject (string): Subject line for email.
+            to (list): List of recipients(string).
+            sender (string): From field for email.
+            body (string): Email body.
+
+        """
+        self.host = host
+        self.port = port
+        self.subject = subject
+        self.to = to
+        self.sender = sender
+        self.body = body
+
+    def write(self, record):
+        """Add record to a buffer to sent via email.
+
+        This method does nothing for now. However, this method
+        show keep adding all the records to the buffer to
+        send via email to particular recipient.
+        """
```

Comment:
The implementation of the `write()` method needs to be completed such that the `done()` method can send emails successfully in order for this pull request to be a minimum viable plugin. We will be merging this pull request after this pull request is functional and this plugin can send emails.

Reply:
Added the implementation in the `write()` method and also wrote proper doc string for that.